metadata-service/factories/src/main/resources/application.yml

```diff
@@ -5,7 +5,7 @@ authentication:
 
   # Required if enabled is true! A configurable chain of Authenticators
   authenticators:
-      # Required for authenticating requests with DataHub-issued Access Tokens - best not to remove.
+    # Required for authenticating requests with DataHub-issued Access Tokens - best not to remove.
```

Comment:
Is this change intended?

Reply:
Automatic reformatting to be consistent with the rest of the file